From: Punit Agrawal <punit@debian.org>
Date: Sat, 24 Feb 2018 08:27:16 +0000
Subject: Fix build failure with golang-1.10

direnv tests fails when built from sources with golang-1.10. The
failure seems to be caused by incorrect escaping of '%' in 'direnv
stdlib'.

Fix this by correcting the escape sequence.
---
 stdlib.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/stdlib.go b/stdlib.go
index acd244b..26b07e6 100644
--- a/stdlib.go
+++ b/stdlib.go
@@ -568,7 +568,7 @@ const STDLIB = "#!bash\n" +
 	"\n" +
 	"    # Strip possible \"/\" suffix from $NODE_VERSIONS, then use that to\n" +
 	"    # Strip $NODE_VERSIONS/$NODE_VERSION_PREFIX prefix from line.\n" +
-	"    while IFS= read -r line; do echo \"${line#${NODE_VERSIONS%/}/${node_version_prefix}}\"; done |\n" +
+	"    while IFS= read -r line; do echo \"${line#${NODE_VERSIONS%%/}/${node_version_prefix}}\"; done |\n" +
 	"\n" +
 	"    # Sort by version: split by \".\" then reverse numeric sort for each piece of the version string\n" +
 	"    sort -t . -k 1,1rn -k 2,2rn -k 3,3rn |\n" +
